### PR TITLE
Login

### PR DESCRIPTION
**Pull request by quansenB**
# Description

Please include a summary of the change and a link to which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
Fixes # (issue)

I added thelogin functionality. It is not yet correctly hooked up with the api endpoint.
I merged with Nabeelahs sign Up, both are working in tandem now. Routing needs to be added when the landing page is ready.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

## Change status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

Please describe steps taken to ensure this feature is functional and does not break other features:
Has been checked with react-logger and checked if working in tandem with signup 
# Checklist:

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation, if relevant
- [x] I have run the app using my feature and ensured that no functionality is broken
- [x] My changes generate no new warnings
- [x] There are no merge conflicts

### CODE REVIEW

File: src/reducers/loginReducer.js [hunk 1]
```diff
@@ -2,12 +2,12 @@ import * as types from "../actions/actionTypes.js";
 
 const initialState_login = {
   token: "",
-  loginSuccess: null,
+  loginSuccess: true,
```
Comment (NabeelahY) [inline src/reducers/loginReducer.js]: I was wondering, shouldn't it be false at first... When the request is successful, then it becomes true?

Comment (NabeelahY) [reply src/reducers/loginReducer.js]: the loginSuccess

Comment (quansenB) [reply src/reducers/loginReducer.js]: It's just used to indicate whether I need to display a "Wrong username or password" message, so true is good for this usecase.

The functionality you are referring to is basically done by the toekn

Comment (NabeelahY) [reply src/reducers/loginReducer.js]: Oh ok then... looks good

**Pull request merged**